schemas/common.py

```
@@ -0,0 +1,22 @@
+
+class Pagination(BaseModel):
+    page: int
+    limit: int
+    total: int
+
+def validate_password_logic(v: str) -> str:
+    """
+    공통 비밀번호 정책 로직 (예: 8자 이상, 특수문자 포함 등)
+    """
+    if len(v) < 8:
+        raise ValueError("비밀번호는 최소 8자 이상이어야 합니다.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)